omg/entities/player.py

```diff
@@ -51,26 +55,43 @@ def __init__(self, name, char_class, image_file, scale, initial_angle=0):
         self.mana_regen_cooldown = 0
 
         # Skills
-        self.skills = SkillManager(N_SKILLS_MAX)
+        self.element = SkillManager(N_SKILLS_MAX)
         self.item_pickup_radius = 5
+        # Initialize an empty element_cash
+        self.to_be_combined_element_cash = []
```

Comment:
cash? --> Did you mean `cache`? If so, I think the buffer might be a better name as you keep some element in this buffer before using it.